Python/Product/Analysis/SaveAnalysis.cs

```diff
@@ -46,14 +53,23 @@ class SaveAnalysis {
                 ModuleInfo moduleInfo;
                 if ((moduleInfo = modKeyValue.Value.Module as ModuleInfo) != null) {
                     _curModule = moduleInfo;
-                    var info = SerializeModule(moduleInfo);
-                    WriteModule(outDir, name, info, moduleInfo.Scope.AllVariables.Keys());
+                    _errorStack.Clear();
+                    _errorStack.Push(modKeyValue.Key);
+                    try {
+                        var info = SerializeModule(moduleInfo);
+                        WriteModule(outDir, name, info, moduleInfo.Scope.AllVariables.Keys());
+                    } catch (Exception ex) {
+                        AddError(ex.ToString());
+                        _errorStack.Clear();
```

Comment:
Seems unnecessary, since the next iteration of the loop will clear before it writes the module.